experimental/gittuf/policy.go

```diff
@@ -10,14 +10,16 @@ import (
 	"log/slog"
 	"strings"
 
+	rslopts "github.com/gittuf/gittuf/experimental/gittuf/options/rsl"
 	"github.com/gittuf/gittuf/internal/policy"
 	"github.com/gittuf/gittuf/internal/rsl"
 	"github.com/gittuf/gittuf/internal/tuf"
 )
 
 var (
-	ErrPushingPolicy = errors.New("unable to push policy")
-	ErrPullingPolicy = errors.New("unable to pull policy")
+	ErrPushingPolicy     = errors.New("unable to push policy")
+	ErrPullingPolicy     = errors.New("unable to pull policy")
+	ErrNoRemoteSpecified = errors.New("no remote specified to commit policy")
```

Comment:
nit: this may be more accurate?

```suggestion
	ErrNoRemoteSpecified = errors.New("no remote specified to push policy to")
```